### DISCUSSION TITLE
graphql/codegen/docs/advanced/generated-files-colocation

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/graphql-code-generator#10092] [Docs Discussions]
# graphql/codegen/docs/advanced/generated-files-colocation

Generate types near your GraphQL operations. Colocate your generated files with your original GraphQL operations.

https://the-guild.dev/graphql/codegen/docs/advanced/generated-files-colocation

**Comment by simplenotezy** [comment]
How do we achieve something similar but using apollo?
